tests/test_stp_global.rb

```diff
@@ -0,0 +1,533 @@
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/stp_global'
+
+# TestX__CLASS_NAME__X - Minitest for X__CLASS_NAME__X node utility class
+class TestStpGlobal < CiscoTestCase
+  # TESTS
+
+  DEFAULT_NAME = 'default'
+
+  def setup
+    super
+    config 'no spanning-tree mode'
+    config 'system bridge-domain none' if n7k_platform?
+  end
+
+  def teardown
+    config 'no spanning-tree mode'
+    config 'system bridge-domain none' if n7k_platform?
+    super
+  end
+
+  def n7k_platform?
+    /N7/ =~ node.product_id
+  end
+
+  def n9k_platform?
+    /N(3|9)/ =~ node.product_id
+  end
+
+  def n6k_platform?
+    /N(5|6)/ =~ node.product_id
+  end
+
+  def create_stp_global(name=DEFAULT_NAME)
+    StpGlobal.new(name)
+  end
+
+  def test_bd_forward_time_change
+    skip('Platform does not support this property') if n6k_platform? ||
+                                                       n9k_platform?
+    @global = create_stp_global
+    config 'system bridge-domain all'
+    bdft = [['2-4,6,8-12', '4'], %w(14 30)]
+    @global.bd_forward_time = bdft
+    assert_equal(bdft, @global.bd_forward_time)
+    bdft = [['2-42', '26'], ['83-92,1000-2300', '20'],
+            ['3000-3960', '16']]
+    @global.bd_forward_time = bdft
+    assert_equal(bdft, @global.bd_forward_time)
+    @global.bd_forward_time = @global.default_bd_forward_time
```

Comment:
Please add an additional default check at the beginning of the test case right after you call `StpGlobal.new`.  Do this for all applicable properties.